ocf_datapipes/consts.py

```diff
@@ -0,0 +1,168 @@
+"""Constants and Enums."""
```

Comment:
should batch key be in constants? Seems more like it is something to do with Batch